src/pages/sign-in/page.jsx

```diff
@@ -31,6 +31,7 @@ const SignInPage = () => {
   const [showPassword, setShowPassword] = useState(false);
   const [forgotPasswordEmail, setForgotPasswordEmail] = useState("");
   const [open, setOpen] = useState(false);
```

Comment:
rename this to `showResetPasswordModal`

Reply:
I don't think it's needed because the state you are refering to is bieng used to control wether the password is visible or not.
<Input
        type={showPassword ? "text" : "password"} // here
        label="Password"
        variant="standard"
        error={!!errors.password}
        helperText={errors.password ? errors.password.message : ""}
        value={value}
        onChange={onChange}
        onBlur={onBlur}
        inputRef={ref}
        fullWidth={true}
        />